catalogue_pipeline/terraform/vhs_sourcedata.tf

```diff
@@ -3,4 +3,6 @@ module "vhs_sourcedata" {
 
   name              = "SourceData"
   table_name_prefix = ""
+
+  table_write_max_capacity = 750
```

Comment:
750 seems a good idea if we've maxed out. I think it's ~ 3 million writes per hour, so we should be able to reindex in about an hour?  SourceData currently has about 2 million entries -- does that seem about right?